soccer/Processor.cpp

```diff
@@ -386,6 +386,16 @@ void Processor::run() {
             delete packet;
         }
 
+        // Log referee data
+        vector<NewRefereePacket*> refereePackets;
+        _refereeModule.get()->getPackets(refereePackets);
+        fprintf(stderr, "Ref packets received this loop:%lu\n",
+                refereePackets.size());
+        for (NewRefereePacket* packet : refereePackets) {
+            SSL_Referee* log = _state.logFrame->add_raw_refbox();
+            log->CopyFrom(packet->wrapper);
+        }
```

Comment:
Actually, I think I see it now.

We make the NewRefPackets with new, then add them to the vector, and in getFrames, we just flush the pointers from the vector, but we should also delete the objects there as well. I'll fix that.

I suspect that vision frames have the same memory leak in this case.